supporting_scripts/generate_code_cov_table/generate_code_cov_table.py

```diff
@@ -73,30 +73,40 @@ def get_changed_files(branch_name, base_branch_name="origin/develop"):
             branch_head = repo.commit(branch_name)
 
     branch_base = repo.merge_base(branch_name, base_branch_name)[0]
-    diff_index = branch_head.diff(branch_base, create_patch=False)
-    file_names = [item.a_path for item in diff_index]
+    diff_index = branch_head.diff(branch_base, create_patch=False, R=True)
 
-    return file_names
+    # File changes with change type
+    file_changes = {}
+    for diff in diff_index:
+        file_path = diff.a_path if diff.a_path else diff.b_path
+        file_changes[file_path] = {
+            'A': 'added',
+            'D': 'deleted',
+            'M': 'modified',
+            'R': 'renamed'
+        }.get(diff.change_type, 'unknown')
 
+    return file_changes
 
-def filter_files(file_names):
-    client_file_names = []
-    server_file_names = []
 
-    for file_name in file_names:
+def filter_file_changes(file_changes):
+    client_file_changes = {}
+    server_file_changes = {}
+
+    for file_name, change_type in file_changes.items():
         if file_name.startswith("src/main/webapp/app"):
             if file_name.endswith(".ts") and "module.ts" not in file_name:
```

Comment:
Not really the subject of this PR, but I just noticed this condition and wondered why the `module.ts` part is needed 🤔 Also, could it be this then?
```suggestion
            if file_name.endswith(".ts") and not file_name.endswith("module.ts"):
```

Reply:
I guess we can add this ^^, I'm adding some other check as well. This is more reliable. The module files never seem to have an associated test file as far as I could see, that is why I ignore them.